packages/cli/src/executable/AgenticaStart.ts

```diff
@@ -34,53 +44,51 @@ export namespace AgenticaStart {
       a.name.localeCompare(b.name),
     );
 
-    const questions = getQuestions({ services: availableServices });
+    const questions = getQuestions({ services: availableServices, options });
     const answers = await inquirer.prompt(questions);
 
-    const validAnswers = typia.assert<{
-      packageManager: "npm" | "yarn" | "pnpm";
-      openAIKey: string;
-      // projectType: string;
-      services: string[];
-    }>(answers);
+    const updatedAnswers = {
+      ...answers,
+      // options
+      projectType: options.project ?? answers.projectType,
+    };
 
-    const {
-      packageManager,
-      openAIKey,
-      // projectType,
-      services,
-    }: {
-      packageManager: "npm" | "yarn" | "pnpm";
+    const validAnswers = typia.assert<{
+      packageManager: PackageManager;
       openAIKey: string;
       services: string[];
-    } = validAnswers;
-
-    // Create project folder
-    const { projectPath } = createProjectDirectory({ projectName });
+      // Options
+      projectType: ProjectOptionValue;
+    }>(updatedAnswers);
 
-    // Create package.json (without dependencies)
-    Package.create({ projectName, projectPath });
+    const { packageManager, openAIKey, projectType, services } = validAnswers;
 
-    // Create tsconfig.json
-    Tsconfig.create({ projectPath });
+    if (projectType === "standalone") {
+      await AgenticaStartOption.Project.execute("standalone")({
+        projectName,
+        projectPath,
+        openAIKey,
+        services,
+      });
+    } else if (projectType === "nodejs") {
```

Comment:
```suggestion
    } else {
      await clone(projectType, projectName);
      await AgenticaStartOption.Project.execute(projectType);
    }
```

and 

```ts
throw new Error(`Invalid project type: ${option}`);
```          
to 
```ts
throw new Error(`Not supported project type: ${option}`);
```